renderer/components/Onboarding/Steps/components/SeedWordList.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import bip39 from 'bip39-en'
+import { Box, Flex } from 'rebass/styled-components'
+import { Text } from 'components/UI'
+
+/**
+ * Create a mapping of first letter to words in BIP 39 list so
+ * we don't have to filter through the whole array when looking up values
+ * every time
+ */
+const bip39WordMap = bip39.reduce((bip39WordMap, word) => {
+  const [firstLetter] = word
+  if (!bip39WordMap[firstLetter]) {
+    bip39WordMap[firstLetter] = []
+  }
+
+  bip39WordMap[firstLetter].push(word)
+
+  return bip39WordMap
+}, {})
+
+const StyledOptionList = React.forwardRef(({ ...rest }, ref) => (
```

Comment:
Is there are reason to use `..rest` if no props were cherry picked? Consider just going with `props`